frontend/packages/console-app/src/components/dashboards-page/OperatorStatus.tsx

```diff
@@ -0,0 +1,228 @@
+  DashboardsOperatorStatus,
+  OperatorStatusPriority,
+  OperatorTitleProps,
+  GetOperatorsWithStatuses,
+} from '../../extensions/dashboards';
+import { getOperatorsHealthState, getOperatorStatusPriority } from './status';
+import { OperatorHealthPriority } from './utils';
+
+import './operator-status.scss';
+
+const OperatorStatusColumn: React.FC<OperatorStatusColumnProps> = ({
+  operator,
+  status,
+  titleLoader,
+}) => {
+  const { title, icon } = status;
+  return (
+    <div className="co-operator-status__row">
+      <AsyncComponent
+        operator={operator}
+        loader={titleLoader}
+        className="co-operator-status__title"
+      />
+      <div className="co-operator-status">
+        <div className="text-secondary">{title}</div>
+        <div className="co-operator-status__icon">{icon}</div>
+      </div>
+    </div>
+  );
+};
+
+const getClusterOperatorStatuses: GetOperatorsWithStatuses = (resources) => {
+  const clusterOperators = resources.clusterOperators?.data ?? [];
+  return _.castArray(clusterOperators).map((co) => {
+    return {
+      operator: co,
+      status: getOperatorStatusPriority(co as ClusterOperator),
+    };
+  });
+};
+
+const OperatorStatuses: React.FC<OperatorStatusesProps> = ({
+  resources,
+  getOperatorsWithStatuses,
+  title,
+  linkTo,
+  titleLoader,
+}) => {
+  const operators = getOperatorsWithStatuses(resources);
+  const mostImportantStatus = Math.max(...operators.map(({ status }) => status.priority));
+  const mappedOperators = operators
+    .filter(({ status }) => status.priority === mostImportantStatus)
+    .sort((a, b) => a.operator.metadata.name.localeCompare(b.operator.metadata.name))
+    .slice(0, 5);
+  return (
+    <>
+      <div className="co-operator-status__row">
+        <div>
+          <span className="co-operator-status__text--bold">{title}</span>
+          <span className="text-secondary">{` (${operators.length})`}</span>
+        </div>
+        <div className="text-secondary">Status</div>
+      </div>
+      {_.values(resources).some((r) => r.loadError) ? (
+        <div className="text-secondary">Not Available</div>
+      ) : (
+        mappedOperators.map(({ operator, status }) => (
+          <OperatorStatusColumn
+            key={operator.metadata.uid}
+            operator={operator}
+            status={status}
+            titleLoader={titleLoader}
+          />
+        ))
+      )}
+      <Link to={linkTo}>View all</Link>
+    </>
+  );
+};
+
+const ClusterOperatorLink: React.FC<OperatorTitleProps> = ({ operator, className }) => (
+  <ResourceLink
+    kind={referenceForModel(ClusterOperatorModel)}
+    name={operator.metadata.name}
+    hideIcon
+    className={className}
+  />
+);
+
+const OperatorStatusPopup: React.FC<OperatorStatusPopupProps> = ({
+  resources,
+  additionalOperators,
+}) => (
+  <>
+    <div className="co-operator-section">
+      Operators extend Kubernetes with additional custom resources to manage applications
```

Comment:
```suggestion
      Operators extend OpenShift with additional custom resources to manage applications.
```

Added a period and switched to "OpenShift". We might want to validate the language here with an SME.

Reply:
@dmesser @tlwu2013 I spoke with @alecmerdler and he mentioned we are transforming this CSV status in the Installed Operator details to show as just "Succeeded." Could that be appropriate here as well? This is a roll-up status of all the CSVs' statuses for an operator across namespaces, showing the "worst" status.

Reply:
fixed